frame/contracts/src/weights.rs

```diff
@@ -154,1339 +152,1311 @@ pub trait WeightInfo {
 	}
 	fn seal_value_transferred(r: u32, ) -> Weight {
-		(138_569_000 as Weight)
-			// Standard Error: 76_000
-			.saturating_add((237_016_000 as Weight).saturating_mul(r as Weight))
+		(129_299_000 as Weight)
+			// Standard Error: 82_000
+			.saturating_add((227_118_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
 	fn seal_minimum_balance(r: u32, ) -> Weight {
-		(134_713_000 as Weight)
-			// Standard Error: 81_000
-			.saturating_add((237_962_000 as Weight).saturating_mul(r as Weight))
+		(126_120_000 as Weight)
+			// Standard Error: 114_000
+			.saturating_add((227_326_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
 	fn seal_tombstone_deposit(r: u32, ) -> Weight {
-		(131_523_000 as Weight)
-			// Standard Error: 90_000
-			.saturating_add((237_435_000 as Weight).saturating_mul(r as Weight))
+		(130_934_000 as Weight)
+			// Standard Error: 89_000
+			.saturating_add((226_638_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
 	fn seal_rent_allowance(r: u32, ) -> Weight {
-		(141_574_000 as Weight)
-			// Standard Error: 86_000
-			.saturating_add((238_102_000 as Weight).saturating_mul(r as Weight))
+		(128_738_000 as Weight)
+			// Standard Error: 77_000
+			.saturating_add((227_062_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
 	fn seal_block_number(r: u32, ) -> Weight {
-		(140_240_000 as Weight)
-			// Standard Error: 101_000
-			.saturating_add((236_568_000 as Weight).saturating_mul(r as Weight))
+		(132_375_000 as Weight)
+			// Standard Error: 88_000
+			.saturating_add((226_861_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
 	fn seal_now(r: u32, ) -> Weight {
-		(138_265_000 as Weight)
-			// Standard Error: 91_000
-			.saturating_add((237_187_000 as Weight).saturating_mul(r as Weight))
-			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
-	}
-	fn seal_rent_params(r: u32, ) -> Weight {
-		(149_701_000 as Weight)
-			// Standard Error: 297_000
-			.saturating_add((357_149_000 as Weight).saturating_mul(r as Weight))
-			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
-	}
-	fn seal_rent_status(r: u32, ) -> Weight {
-		(146_863_000 as Weight)
-			// Standard Error: 191_000
-			.saturating_add((638_683_000 as Weight).saturating_mul(r as Weight))
```

Comment:
@athei where did these benchmarks go?

Reply:
Those two are functions marked as unstable. I removed the benchmarks for now but they are re-added once the API is stabilized (or before that). Unstable APIs cannot be used in production chains and are only available when you supply the`unstable-interface` feature to pallet_contracts.

So this is OK.